server/py/services/api/daemon.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import framework.service
+import services.api.main
+
+
+class Daemon(framework.service.Daemon):
+    def __init__(self, service_cls: framework.service.Service.__class__):
+        self._service = service_cls()
+
+    def initialize(self):
+        self._service.initialize()
+
+
+daemon = Daemon(service_cls=services.api.main.Service)
+daemon.initialize()
+app = daemon.app
```

Comment:
```suggestion
if __name__ == "__main__":
  daemon = Daemon(service_cls=services.api.main.Service)
  daemon.initialize()
  app = daemon.app
```

Reply:
This doesn't work with our current uvircorn command. May be able to get it to work with the new fastapi CLI but it requires some investigation